### DISCUSSION TITLE
Submitting new Program

### DISCUSSION THREAD

**Discussion by UNiXMIT** [microsoft/winget-create#324] [Q&A]
When you submit a new program to be included in Winget, do you have to be the owner/creator/company of that program? Am I allowed to submit other people’s programs that I want to be included in Winget? Thanks

**Comment by Masamune3210** [comment]
No, you can submit any program as long as it fits the criteria of requirements and isn't on a very small list of apps that have been asked to be removed. Make sure you look at pending issues and pull requests first though as to avoid duplicates please
